src/reducers/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { combineReducers } from "redux";
+import { DifficultyLevel, screenInfo } from "../helpers/enums";
+
+const currentUserScreen = (store, action) => {
+  if (action.type === "USER_SCREEN") {
+    return {
+      value: action.value,
+    };
+  }
+  return store || { value: screenInfo.HOME };
+};
+const userProperties = (store, action) => {
+  if (action.type === "USER_PROPERTIES") {
+    return action.value;
+  }
+  return store || { difficultyLevel: DifficultyLevel.EASY, userName: "YOLO" };
```

Comment:
Do not default player name.
The player name field is mandatory. If the player doesn't fill this field, show a proper error message.